README.md

```diff
@@ -4,6 +4,18 @@ A minimal MuJoCo simulation with ROS communication.
 ## Usage
 Inside a fresh catkin workspace, clone this repo inside `/path/to/ws/src/`. Then, run `catkin_make`.
 
+### Mujoco
+Download, compile and install mujoco:
+```
+cd MUJOCO/PATH
```

Comment:
Nit: I prefer `/path/to/mujoco` to avoid confusion with environment variables like `$PATH`.